src/app/config.rs

```diff
@@ -77,6 +77,8 @@ pub struct Config {
     /// a new connection.
     pub control_backoff_delay: Duration,
 
+    pub control_connect_timeout: Duration,
```

Comment:
Nit: It seems like most of the other fields here have doc comments, can we get one here as well>